README.md

```diff
@@ -1 +1,44 @@
 # Narwhal!
+
+[![GoDoc shield](https://raw.githubusercontent.com/golang/gddo/c782c79e0a3c3282dacdaaebeff9e6fd99cb2919/gddo-server/assets/status.svg)][docs]
+
+Narwhal is a Go API for Docker and Docker containers.
+
+## Examples
```

Comment:
These would be better as [examples](https://pkg.go.dev/testing?tab=doc#hdr-Examples) since they show up in the reference documentation and are tested to ensure they compile. The convention is to use a file `example_test.go` in the same package, but start it with `package narwhal_test` to force the code to import the package.